spec/setup_test_database.rb

```diff
@@ -0,0 +1,8 @@
+require 'pg'
+
+p "Setting up test database..."
+
+def setup_test_database
+  connection = PG.connect(dbname: 'bookmark_manager_test')
```

Comment:
It might be a good idea to take the conditional for the connection in the model chitter.rb, and place it into here, since this is where we're dealing with that stuff. 
```
if ENV['ENVIRONMENT'] == 'test'
      connection = PG.connect(dbname: 'chitter_test')
    else
      connection = PG.connect(dbname: 'chitter')
    end
```